redact.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Redact Enterprise Suite
+After=docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Environment="INSTALLATION_DIR=/etc/redact"
+ExecStartPre=/bin/bash -c '${INSTALLATION_DIR}/stop_redact.sh'
+ExecStart=/bin/bash -c '${INSTALLATION_DIR}/start_redact.sh'
```

Comment:
I'm unsure if for starting it is better to do for ExecStart:
`docker compose -f $installation_dir/docker-compose.yaml up --force-recreate -d --remove-orphans ${services}`
When I created the systemd service for RLS, I had it in the beginning in a similar way - also using type `simple`. 
Georg then advised to attach to STDIN and STDOUT so systemd can treat it as a process. Look [here](https://github.com/brighter-ai/redact-license-server/blob/main/scripts/systemd/rls.service).

This would then lead to some more changes. Maybe let's talk how you tested.

Reply:
Removing the `-d` option from docker compose seems to do the trick. I added it as an option for `start_redact.sh`, with a note that it's recommended when running the script directly.